helm/stat/Chart.yaml

```diff
@@ -1,15 +1,15 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
+apiVersion: v2
+description: A hydro-stat service helm chart
```

Comment:
A helm chart for the stat service which provides statistical report about probable data/concept drift